File: Python/Normal-FizzBuzz.py
#A simple Python code to implement FizzBuzz
#Author @pranay-ar

# this for loop is run in range 0 to 100 because we know for loops is allways run from 0 not 1 and end with the  one digit befor 

for fizzbuzz in range(101):  ## for fizzbuzz in range(0,101):
    if fizzbuzz % 3 == 0 and fizzbuzz % 5 == 0:
        print("fizzbuzz")
        continue
    elif fizzbuzz % 3 == 0:
        print("fizz")
        continue
    elif fizzbuzz % 5 == 0:
        print("buzz")
        continue
    print(fizzbuzz)

File: C++/normal_fizzbuzz.cpp
//Normal Fizzbuzz solution in C++
//author: blackfly19

#include<bits/stdc++.h>
using namespace std;

int main()
{
	for(int i=0;i<=100;i++)
	{
		if(i%3 == 0 && i%5 == 0)
			cout<<"fizzbuzz";
		else
		if(i%5 == 0)
			cout<<"buzz";
		else
		if(i%3 == 0)
			cout<<"fizz";
		else
			cout<<i;
		cout<<"\n";
	}
}
